onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -155,6 +158,12 @@ def __next__(self):
             raise StopIteration
         return result
 
+    def __len__(self):
+        raise NotImplementedError
+
+    def set_range(self, start_index: int, end_index: int):
```

Comment:
I would suggest adding documentation about what implementers of `CalibrationDataReader` should do with this function. If I understand correctly, this tells the data reader that calls to `get_next()` should only return data between the start and end indices.

Reply:
bump